### PR TITLE
Fixes the Operating Computer.

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request
- Fixes the popout bug.
  https://github.com/yogstation13/-tg-station-yog-rebase/pull/74
#### Super3222

:cl:
bugfix: Operating Computer's pop out bug has been fixed.
/:cl:

### CODE REVIEW

**Comment by oisin100** [comment]
rscadd = bugfix?
I think not.

**Comment by Super3222** [comment]
All that porting messed with my head

**Comment by oisin100** [comment]
Also, Try updating the code to the code you did when you ported to /TG/

File: code/game/machinery/computer/Operating.dm [hunk 1]
```diff
@@ -7,8 +7,21 @@
 	var/mob/living/carbon/human/patient = null
 	var/obj/structure/optable/table = null
 	var/menu = 1 // one is the standard surgery set up two is scanning cubicles, implant cases, and possibly diskettes, three is database.
-	var/implants = list()
-	var/organs = list()
+	var/list/implants = list(
+		/obj/item/weapon/implant/mindshield,
```
Comment (oisin100) [inline code/game/machinery/computer/Operating.dm]: We dont have dem silly mindshield implants on regular yogs.

File: code/game/machinery/computer/Operating.dm [hunk 2]
```diff
@@ -50,11 +50,11 @@
 
 /obj/machinery/computer/operating/proc/bootup_organs()
 	audible_message("[src] reboots their organ database and reverts back the intiial set.")
```
Comment (Cruix) [inline code/game/machinery/computer/Operating.dm]: I thiink there is somethiing wrong wiith thiis liine.

**Pull request merged**